scripts/installPackages.js

```diff
@@ -1,14 +1,20 @@
 // install_packages.js
 // made by Homura
+// refactored by LeonardoBagi
 const chalk = require("chalk");
-const fs = require("fs");
 const childProcess = require("child_process");
 
 const package = require("../package.json");
-const missing = Object.keys(package.dependencies)
-    .filter((package) => !fs.existsSync(`../node_modules/${package}`));
+const missing = Object.keys(package.dependencies).filter(package => {
```

Comment:
I don't see how a brute forced try/catch is more efficient than checking if the module exists

Reply:
That other code was always saying that the package didn't exist tho.. `require.resolve()` will only return the file name of the module so it's better to check it via that than to look into `node_modules` as far as I know